src/routes/controllers/articles.controller.js

```diff
@@ -126,23 +130,31 @@ export async function removeArticle(req, res) {
  * @returns {Object} res with 404 response if the array is empty
  */
 export async function getArticle(req, res) {
-  const { articleId } = req.params;
+  const articleSlug = req.params.slug;
+  let requestUser;
+  const token = req.headers.authorization || req.headers['x-access-token'];
+
```

Comment:
I think there is middleware handling this token authorization yeah? why not use that instead of verifying the token inside the controller

Reply:
You know the `checkAuthorizedUser` middleware throws you back if you don't provide a valid token right? It means if i apply it a public route like `get a single article`, unauthenticated users won't be able to read articles. That is why i didn't use it.

_N.B. It seems like you clicked on `request change` when you wanted to submit your review?_

Reply:
Oh ok I get now, I think we should tweak the `verifyToken` helper function to contain the `token` variable so we don't always have to declare it anytime we need to verify a token

Reply:
I am glad you understand now. 
BTW, we should expect a PR from you to that effect right?